### DISCUSSION TITLE
{Help] Postal V2 Not Able to Check Run CLI -> postal status/postal start/postal stop

### DISCUSSION THREAD

**Discussion by XXXLukieDukieXXX** [postalserver/postal#3167] [Help with using Postal]
Anyone of you running postal v2 version, are you guys able to run the command for postal status/postal start/postal stop?
Below is the result, I recalled previously I able to run the command. Not sure why it has been changed. Help me please.

root@spmail:/opt/postal/config# postal status
Usage: postal [-m maximum-message-size] [-M minimum-message-size] [-t threads]
              [-c messages-per-connection] [-r messages-per-minute] [-a]
              [-b [no]netscape] [-p port] [-[z|Z] debug-file]
              [-s ssl-percentage]
              [-L] [-l local-address] [-f sender-file]
              smtp-server user-list-filename

Postal Version: 0.75
root@spmail:/opt/postal/config# postal stop
Usage: postal [-m maximum-message-size] [-M minimum-message-size] [-t threads]
              [-c messages-per-connection] [-r messages-per-minute] [-a]
              [-b [no]netscape] [-p port] [-[z|Z] debug-file]
              [-s ssl-percentage]
              [-L] [-l local-address] [-f sender-file]
              smtp-server user-list-filename

Postal Version: 0.75
root@spmail:/opt/postal/config# postal start
Usage: postal [-m maximum-message-size] [-M minimum-message-size] [-t threads]
              [-c messages-per-connection] [-r messages-per-minute] [-a]
              [-b [no]netscape] [-p port] [-[z|Z] debug-file]
              [-s ssl-percentage]
              [-L] [-l local-address] [-f sender-file]
              smtp-server user-list-filename

Postal Version: 0.75
root@spmail:/opt/postal/config#

**Comment by willpower232** [comment]
Are you running it as a docker image? Did you follow the regular install instructions? What is the output of `which postal`?
